recipe/meta.yaml

```diff
@@ -56,6 +56,11 @@ outputs:
         - {{ pin_subpackage("libllvm" + major_ver, exact=True) }}
         - {{ pin_subpackage("llvm-tools", exact=True) }}
         - libcxx >={{ cxx_compiler_version }}.a0                        # [osx]
+      run_constrained:
+        - llvm        {{ version }}
+        - llvm-tools  {{ version }}
+        - clang       {{ version }}
+        - clang-tools {{ version }}
```

Comment:
Adding `llvm-tools` as a run-constraint when it's already pinned as a subpackage seems... unnecessary?